git-hooks/post-checkout

```diff
@@ -1,3 +1,2 @@
-#!/usr/bin/env sh
-
-tar -C ./Carthage -xzf ./Carthage/Build.tar.gz
+#!/usr/bin/env bash
+exec git submodule update
```

Comment:
do i need to do `--init` as well? i wouldn't have thought so. you can't use ios-tools to initialise itself!